### ISSUE TITLE
Zooming in list view shouldn't affect text columns

### ISSUE THREAD

**Issue by memeplex** [linuxmint/nemo#1886]
```
~:: uname -a
Linux carlos 4.14.52-1-MANJARO #1 SMP PREEMPT Tue Jun 26 15:24:31 UTC 2018 x86_64 GNU/Linux

~:: cinnamon --version
Cinnamon 3.8.6

~:: nemo --version
nemo 3.8.3

~:: lspci | grep VGA
00:02.0 VGA compatible controller: Intel Corporation 2nd Generation Core Processor Family Integrated Graphics Controller (rev 09)

~:: pacman -Qs xf86-video-intel
local/xf86-video-intel 1:2.99.917+829+gd7dfab62-1 (xorg-drivers)
    X.org Intel i810/i830/i915/945G/G965+ video drivers
 ```
In list mode incrementing the zoom level makes the icons larger, as expected, but also the text in the other columns (size, type, date, etc). This is not subtle at all, the text is quickly enlarged to a bizarrely big size. At the third zoom level text in the right pane is already way bigger that text in left pane (I mean the main one, with the overall tree view).

**Comment by Deduction42** [comment]
I can confirm this is happening. Oddly enough the text size doesn't change when zoom levels are changed in icon or compact view. List view should exhibit the same behaviour (where the text size is set by the fonts category under system settings).

**Comment by Deduction42** [comment]
Is there any plan to make the text size in list view be determined by the overall text settings and not the zoom? I find it very annoying that the text gets bigger if I need bigger icons.

**Comment by OliverLew** [comment]
I second the request to keep the size of text in list view. It's just too inconsistent with the whole application.

**Comment by marksolaris** [comment]
I use this to enforce font size. You can choose a smaller size, but this is how it's done.
https://github.com/linuxmint/nemo/issues/2250

**Comment by SebiTimeWaster** [comment]
i have to disagree on this one, the list view is not meant to show image previews, it is a list of files that should be consitently sized vertically to ensure that the most amount of items fit the screen, so increasing the zoom should consistently zoom text and icons at the same time (to adapt to high dpi screens for instance, and yes, i know mint has a zoom setting in the screen settings, but not every application consistently supports that setting so i cant use it yet).

if you want big preview icons i would switch to one of the other views. 

what would be ok though is if icons and text can be zoomed independently, though it is more work for the programmers.

**Comment by marksolaris** [comment]
"so increasing the zoom should consistently zoom text and icons at the same time"

In the original bug report, and noticed by everyone, it says "the text is quickly enlarged to a bizarrely big size".

When you do the thing and use the scroll wheel to zoom into the pictures (in any view) you want the very readable text to stay very readable. It's not at all viable for the filename text next to each icon to be 700pt, or shrink to 1pt.

Our ways of working in Icon, List and Compact views have many repeating use cases where zooming with the mouse wheel is required to take a better look at the image icon. What's illogical is shrinking or zooming the text so it's unreadable. That lack of readability serves no useful purpose in subsequent file management.

Anyway I coded myself the fix per #2250 and use it every day.
